azure_monitor/src/azure_monitor/sdk/auto_collection/live_metrics/sender.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+#
+import json
+import logging
+import time
+
+import requests
+
+from azure_monitor.protocol import LiveMetricEnvelope
+
+DEFAULT_LIVEMETRICS_ENDPOINT = "https://rt.services.visualstudio.com"
+LIVE_METRICS_TRANSMISSION_TIME_HEADER = "x-ms-qps-transmission-time"
+
+logger = logging.getLogger(__name__)
+
+
+class LiveMetricsSender:
+    """Live Metrics Sender
+
+    Send HTTP requests to Live Metrics service
+    """
+
+    def __init__(self, instrumentation_key: str):
+        self._endpoint = DEFAULT_LIVEMETRICS_ENDPOINT
```

Comment:
Why do we need `self._endpoint`? Can't we just refer to the global variable since it will never change?